pde_poc.Tests/Engine/MV/ExecutorIntegrationTests.cs

```diff
@@ -0,0 +1,230 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Options;
+
+using Xunit;
+
+using pde_poc_sim.Engine;
+using pde_poc_sim.Engine.Lib;
+
+using pde_poc_sim.OpenFisca;
+
+using RestSharp;
+
+namespace pde_poc.Tests
+{
+    public class ExecutorIntegrationTests
+    {
+        
+        [Fact(Skip="Integration test")]
```

Comment:
Integration tests are skipped on normal test execution, since not a true unit test (has a dependency on open fisca). But good to have in the repo when developing